File: Level 1/Assets/Rebirth.js
var obj : GameObject;
public var Player = new GameObject[5];

function Update() {
	if (Input.GetKeyDown("r")) {

		for (var i = 1; i < 5; i++) {
			if (Player[i].activeSelf) { 
				obj = Player[i];
				if (obj.activeSelf) {obj.SetActive(false); }
			}
		}
	
		Player[0].SetActive(true);
		//Player[0].transformation.position = obj.transformation.position;
	}

}

File: Level 1/Assets/Scripts/Rebirth.cs
using UnityEngine;
using System.Collections;

public class Rebirth : MonoBehaviour {
	public GameObject[] Player;
	public GameObject obj;

	void Update () {
		if (Input.GetKeyDown("r")) {
			for (var i = 0; i < 5; i++) {
				if (Player[i].activeSelf) { 
					obj = Player[i];
					obj.SetActive(false);
					Player[1].transform.position = Player[i].transform.position;
					Player[1].SetActive(true);
					break;
				}
			}
		}
	}

}
